// Copyright (c) 2020 The Orbit Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

package orbit_client_protos;

message PresetModule {
  repeated uint64 function_hashes = 1;
}

message PresetInfo {
  reserved 1;
  map<string, PresetModule> path_to_module = 2;
}

// this message isn't used for serialization
message PresetFile {
  string file_name = 1;
  PresetInfo preset_info = 2;
}
